esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -1172,7 +1172,7 @@ void retrieveQuerysetDetailsFromAlias(IEspContext &context, IPropertyTree *regis
         IPropertyTree *query = registry->queryPropTree(xpath);
         if (!query)
         {
-            UWARNLOG("No matching Query %s found for Alias %s", a->getId(), name);
```

Comment:
This is fixing the symptom, not the bug. I assume the compiler warned that a may be (actually is always, I think) null. But the fix should be to take heed of the warning and trace the id properly, not just add code that makes the warning go away.

Reply:
Opened https://track.hpccsystems.com/browse/HPCC-27812 for closer scrutiny of my "fix"